core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2990,9 +2990,9 @@ class ReplicaManagerTest {
                             requiredAcks: Short = -1): CallbackResult[PartitionResponse] = {
     val result = new CallbackResult[PartitionResponse]()
     val topicIdPartition = new TopicIdPartition(topicId, partition)
-    def appendCallback(responses: Map[TopicIdPartition, PartitionResponse]): Unit = {
-      val response = responses.get(topicIdPartition)
-      assertTrue(response.isDefined)
+    def appendCallback(responses: util.Map[TopicIdPartition, PartitionResponse]): Unit = {
+      val response = java.util.Optional.ofNullable(responses.get(topicIdPartition))
```

Comment:
We already imported `java.util.Optional`, so we don't need a full-qualified name.